webapp/src/main/java/org/apache/atlas/web/security/AtlasSecurityConfig.java

```diff
@@ -96,6 +96,12 @@ public class AtlasSecurityConfig extends WebSecurityConfigurerAdapter {
     private final StaleTransactionCleanupFilter staleTransactionCleanupFilter;
     private final ActiveServerFilter activeServerFilter;
     private final LaunchDarklyConfig launchDarklyConfig;
+    private final static String LAUNCH_DARKLY_SDK_KEY       = System.getenv("USER_LAUNCH_DARKLY_SDK_KEY");
```

Comment:
System.getenv("USER_LAUNCH_DARKLY_SDK_KEY"); - This can give NPE, please use with try catch

Reply:
System.getenv("USER_LAUNCH_DARKLY_SDK_KEY") doesn’t give NPE, just checking if it retun null then keep it empty, will solve the problem